h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -111,17 +116,21 @@ private Leaderboard() {
   /**
    *
    */
-  private Leaderboard(String project_name, UserFeedback userFeedback, Frame leaderboardFrame) {
+  public Leaderboard(String project_name, UserFeedback userFeedback, Frame leaderboardFrame) {
     this._key = make(idForProject(project_name));
     this.project_name = project_name;
     this.userFeedback = userFeedback;
     this.leaderboardFrame = leaderboardFrame;
+
     if (null != this.leaderboardFrame) {
       this.leaderboardFrameChecksum = leaderboardFrame.checksum();
     } else {
       this.leaderboardFrameChecksum = 0;
     }
-    DKV.put(this);
+
+    this.sort_metric = "auc";
+    this.other_metrics = new String[] { "logloss" };
+    this.sort_decreasing = true;
```

Comment:
The above are now "default" sort metrics, etc?
 Just curious as to why these statements are needed?

Reply:
Because an empty leaderboard needs something there so that clients don't have to special case.  As the code was, the R and Python clients both crashed partially because these weren't set.